var/spack/repos/builtin.mock/packages/a/package.py

```diff
@@ -54,6 +54,10 @@ class A(AutotoolsPackage):
 
     depends_on('b', when='foobar=bar')
 
+    @property
```

Comment:
@alalazo I really appreciate the addition of the unit test. In my opinion though it would be worthwhile to create a new mock package for testing. Although this creates some redundancy, I think it avoids potential issues with confusing the purpose of unit tests when multiple tests use the same mock package. If that seems sensible to you then I'll wait for this to be separated out; otherwise let me know and I'll check with other folks internally to see if there's a general consensus (if most folks disagree with me I'll accept this as is).

Reply:
@scheibelp I'll create a new mock package, no worries. Usually I reuse what's there because adding new mock packages requires more lines than I would like.

> If that seems sensible to you then I'll wait for this to be separated out;

:+1: